Committee.md

```diff
@@ -159,24 +139,19 @@ There is no process for members of the public at large to directly add or remove
 
 ## Conflicts of interest
 
-For the sake of transparency, the committee chair is expected to disclose any
-known potential conflict of interest between committee members and proponents of
-a specific language change at the start of the committee deliberation.
+Currently, the committee classifies at least the following situations as being
+unconditional sources of conflicts of interest:
 
-Taking in account the small world nature of the OCaml community, the committee
-considers that transparency is a sufficient measure to avoid unconscious or
-covert influences on the deliberation. Thus conflicted committee members are not
-expected to recuse themselves from the deliberation nor the potential votes.
+1. belonging to the same institution as the proposal submitter
+2. on-going or recent direct financial ties to the proposal submitter
```

Comment:
nit: in case (1), the submitter is a person, but in case (2) it reads like it is an entity (of course we don't mean financial ties with the person submitting the proposal specifically).